addons/link_tracker/controller/main.py

```diff
@@ -5,15 +5,22 @@
 from odoo.http import request
 
 
+ignore_user_agent = ['twitterbot', 'facebookexternalhit', 'facebot', 'applebot', 'bingbot', 'trendsmapresolver'
+                            'livelapbot', 'linkedinbot', 'whatsapp', 'googlebot', 'pinterest', 'instagram']
```

Comment:
indentation issue (are you sure your linter is correctly configured ?)
![screenshot_spect](https://user-images.githubusercontent.com/54316226/143032884-1aea4f6b-813e-4384-a744-e72e9be5b0f3.png)